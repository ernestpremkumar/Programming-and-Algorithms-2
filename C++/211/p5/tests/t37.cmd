1000 0 tests/t37.in results/t37.myout
