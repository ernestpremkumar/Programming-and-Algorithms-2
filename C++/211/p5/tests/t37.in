Homer shopper 1 42000
Marge shopper 1 42000
Lisa shopper 1 42000
Maggie shopper 1 42000
Patty robber 1 42000
Grampa shopper 1 42000
Bart robber 1 42000
Selma shopper 1 42000
Smithers shopper 1 42000
Barney robber 1 42000
Burns shopper 1 42000
Moe robber 1 42000
Rod shopper 1 42000
Maude shopper 1 42000
Todd shopper 1 42000
Apu shopper 1 42000
Wiggum robber 1 42000
Hibbert shopper 1 42000
Ned shopper 1 42000
Krusty robber 1 42000
Sideshow robber 1 42000
Scratchy robber 1 42000
Otto shopper 1 42000
Edna shopper 1 42000
Seymour shopper 1 42000
Itchy robber 1 42000
Willie robber 1 42000
