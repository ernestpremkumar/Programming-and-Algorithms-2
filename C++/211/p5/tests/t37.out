1: Homer entered store
1: Marge entered store
1: Lisa entered store
1: Maggie entered store
1: Patty entered store
1: Grampa entered store
1: Bart entered store
1: Selma entered store
1: Smithers entered store
1: Barney entered store
1: Burns entered store
1: Moe entered store
1: Rod entered store
1: Maude entered store
1: Todd entered store
1: Apu entered store
1: Wiggum entered store
1: Hibbert entered store
1: Ned entered store
1: Krusty entered store
1: Sideshow entered store
1: Scratchy entered store
1: Otto entered store
1: Edna entered store
1: Seymour entered store
1: Itchy entered store
1: Willie entered store
84001: Homer done shopping
84001: Marge done shopping
84001: Lisa done shopping
84001: Maggie done shopping
84001: Patty done shopping
84001: Grampa done shopping
84001: Bart done shopping
84001: Selma done shopping
84001: Smithers done shopping
84001: Barney done shopping
84001: Burns done shopping
84001: Moe done shopping
84001: Rod done shopping
84001: Maude done shopping
84001: Todd done shopping
84001: Apu done shopping
84001: Wiggum done shopping
84001: Hibbert done shopping
84001: Ned done shopping
84001: Krusty done shopping
84001: Sideshow done shopping
84001: Scratchy done shopping
84001: Otto done shopping
84001: Edna done shopping
84001: Seymour done shopping
84001: Itchy done shopping
84001: Willie done shopping
84001: Homer started checkout with checker 0
84001: Marge started checkout with checker 1
84001: Lisa started checkout with checker 2
84001: Maggie started checkout with checker 3
84001: Patty started checkout with checker 4
84001: Grampa started checkout with checker 5
84001: Bart started checkout with checker 6
84001: Selma started checkout with checker 7
84001: Smithers started checkout with checker 8
84001: Barney started checkout with checker 9
84001: Burns started checkout with checker 10
84001: Moe started checkout with checker 11
84001: Rod started checkout with checker 12
84001: Maude started checkout with checker 13
84001: Todd started checkout with checker 14
84001: Apu started checkout with checker 15
84001: Wiggum started checkout with checker 16
84001: Hibbert started checkout with checker 17
84001: Ned started checkout with checker 18
84001: Krusty started checkout with checker 19
84001: Sideshow started checkout with checker 20
84001: Scratchy started checkout with checker 21
84001: Otto started checkout with checker 22
84001: Edna started checkout with checker 23
84001: Seymour started checkout with checker 24
84001: Itchy started checkout with checker 25
84001: Willie started checkout with checker 26
84008: Patty stole $250 and 42000 items from checker 4
84008: Bart stole $250 and 42000 items from checker 6
84008: Barney stole $250 and 42000 items from checker 9
84008: Moe stole $250 and 42000 items from checker 11
84008: Wiggum stole $250 and 42000 items from checker 16
84008: Krusty stole $250 and 42000 items from checker 19
84008: Sideshow stole $250 and 42000 items from checker 20
84008: Scratchy stole $250 and 42000 items from checker 21
84008: Itchy stole $250 and 42000 items from checker 25
84008: Willie stole $250 and 42000 items from checker 26
126001: Homer paid $126000 for 42000 items to checker 0
126001: Marge paid $126000 for 42000 items to checker 1
126001: Lisa paid $126000 for 42000 items to checker 2
126001: Maggie paid $126000 for 42000 items to checker 3
126001: Grampa paid $126000 for 42000 items to checker 5
126001: Selma paid $126000 for 42000 items to checker 7
126001: Smithers paid $126000 for 42000 items to checker 8
126001: Burns paid $126000 for 42000 items to checker 10
126001: Rod paid $126000 for 42000 items to checker 12
126001: Maude paid $126000 for 42000 items to checker 13
126001: Todd paid $126000 for 42000 items to checker 14
126001: Apu paid $126000 for 42000 items to checker 15
126001: Hibbert paid $126000 for 42000 items to checker 17
126001: Ned paid $126000 for 42000 items to checker 18
126001: Otto paid $126000 for 42000 items to checker 22
126001: Edna paid $126000 for 42000 items to checker 23
126001: Seymour paid $126000 for 42000 items to checker 24
registers[0] = $126250
registers[1] = $126250
registers[2] = $126250
registers[3] = $126250
registers[4] = $0
registers[5] = $126250
registers[6] = $0
registers[7] = $126250
registers[8] = $126250
registers[9] = $0
registers[10] = $126250
registers[11] = $0
registers[12] = $126250
registers[13] = $126250
registers[14] = $126250
registers[15] = $126250
registers[16] = $0
registers[17] = $126250
registers[18] = $126250
registers[19] = $0
registers[20] = $0
registers[21] = $0
registers[22] = $126250
registers[23] = $126250
registers[24] = $126250
registers[25] = $0
registers[26] = $0
registers[27] = $250
registers[28] = $250
registers[29] = $250
registers[30] = $250
registers[31] = $250
registers[32] = $250
registers[33] = $250
registers[34] = $250
registers[35] = $250
registers[36] = $250
registers[37] = $250
registers[38] = $250
registers[39] = $250
registers[40] = $250
registers[41] = $250
registers[42] = $250
registers[43] = $250
registers[44] = $250
registers[45] = $250
registers[46] = $250
registers[47] = $250
registers[48] = $250
registers[49] = $250
registers[50] = $250
registers[51] = $250
registers[52] = $250
registers[53] = $250
registers[54] = $250
registers[55] = $250
registers[56] = $250
registers[57] = $250
registers[58] = $250
registers[59] = $250
registers[60] = $250
registers[61] = $250
registers[62] = $250
registers[63] = $250
registers[64] = $250
registers[65] = $250
registers[66] = $250
registers[67] = $250
registers[68] = $250
registers[69] = $250
registers[70] = $250
registers[71] = $250
registers[72] = $250
registers[73] = $250
registers[74] = $250
registers[75] = $250
registers[76] = $250
registers[77] = $250
registers[78] = $250
registers[79] = $250
registers[80] = $250
registers[81] = $250
registers[82] = $250
registers[83] = $250
registers[84] = $250
registers[85] = $250
registers[86] = $250
registers[87] = $250
registers[88] = $250
registers[89] = $250
registers[90] = $250
registers[91] = $250
registers[92] = $250
registers[93] = $250
registers[94] = $250
registers[95] = $250
registers[96] = $250
registers[97] = $250
registers[98] = $250
registers[99] = $250
registers[100] = $250
registers[101] = $250
registers[102] = $250
registers[103] = $250
registers[104] = $250
registers[105] = $250
registers[106] = $250
registers[107] = $250
registers[108] = $250
registers[109] = $250
registers[110] = $250
registers[111] = $250
registers[112] = $250
registers[113] = $250
registers[114] = $250
registers[115] = $250
registers[116] = $250
registers[117] = $250
registers[118] = $250
registers[119] = $250
registers[120] = $250
registers[121] = $250
registers[122] = $250
registers[123] = $250
registers[124] = $250
registers[125] = $250
registers[126] = $250
registers[127] = $250
registers[128] = $250
registers[129] = $250
registers[130] = $250
registers[131] = $250
registers[132] = $250
registers[133] = $250
registers[134] = $250
registers[135] = $250
registers[136] = $250
registers[137] = $250
registers[138] = $250
registers[139] = $250
registers[140] = $250
registers[141] = $250
registers[142] = $250
registers[143] = $250
registers[144] = $250
registers[145] = $250
registers[146] = $250
registers[147] = $250
registers[148] = $250
registers[149] = $250
registers[150] = $250
registers[151] = $250
registers[152] = $250
registers[153] = $250
registers[154] = $250
registers[155] = $250
registers[156] = $250
registers[157] = $250
registers[158] = $250
registers[159] = $250
registers[160] = $250
registers[161] = $250
registers[162] = $250
registers[163] = $250
registers[164] = $250
registers[165] = $250
registers[166] = $250
registers[167] = $250
registers[168] = $250
registers[169] = $250
registers[170] = $250
registers[171] = $250
registers[172] = $250
registers[173] = $250
registers[174] = $250
registers[175] = $250
registers[176] = $250
registers[177] = $250
registers[178] = $250
registers[179] = $250
registers[180] = $250
registers[181] = $250
registers[182] = $250
registers[183] = $250
registers[184] = $250
registers[185] = $250
registers[186] = $250
registers[187] = $250
registers[188] = $250
registers[189] = $250
registers[190] = $250
registers[191] = $250
registers[192] = $250
registers[193] = $250
registers[194] = $250
registers[195] = $250
registers[196] = $250
registers[197] = $250
registers[198] = $250
registers[199] = $250
registers[200] = $250
registers[201] = $250
registers[202] = $250
registers[203] = $250
registers[204] = $250
registers[205] = $250
registers[206] = $250
registers[207] = $250
registers[208] = $250
registers[209] = $250
registers[210] = $250
registers[211] = $250
registers[212] = $250
registers[213] = $250
registers[214] = $250
registers[215] = $250
registers[216] = $250
registers[217] = $250
registers[218] = $250
registers[219] = $250
registers[220] = $250
registers[221] = $250
registers[222] = $250
registers[223] = $250
registers[224] = $250
registers[225] = $250
registers[226] = $250
registers[227] = $250
registers[228] = $250
registers[229] = $250
registers[230] = $250
registers[231] = $250
registers[232] = $250
registers[233] = $250
registers[234] = $250
registers[235] = $250
registers[236] = $250
registers[237] = $250
registers[238] = $250
registers[239] = $250
registers[240] = $250
registers[241] = $250
registers[242] = $250
registers[243] = $250
registers[244] = $250
registers[245] = $250
registers[246] = $250
registers[247] = $250
registers[248] = $250
registers[249] = $250
registers[250] = $250
registers[251] = $250
registers[252] = $250
registers[253] = $250
registers[254] = $250
registers[255] = $250
registers[256] = $250
registers[257] = $250
registers[258] = $250
registers[259] = $250
registers[260] = $250
registers[261] = $250
registers[262] = $250
registers[263] = $250
registers[264] = $250
registers[265] = $250
registers[266] = $250
registers[267] = $250
registers[268] = $250
registers[269] = $250
registers[270] = $250
registers[271] = $250
registers[272] = $250
registers[273] = $250
registers[274] = $250
registers[275] = $250
registers[276] = $250
registers[277] = $250
registers[278] = $250
registers[279] = $250
registers[280] = $250
registers[281] = $250
registers[282] = $250
registers[283] = $250
registers[284] = $250
registers[285] = $250
registers[286] = $250
registers[287] = $250
registers[288] = $250
registers[289] = $250
registers[290] = $250
registers[291] = $250
registers[292] = $250
registers[293] = $250
registers[294] = $250
registers[295] = $250
registers[296] = $250
registers[297] = $250
registers[298] = $250
registers[299] = $250
registers[300] = $250
registers[301] = $250
registers[302] = $250
registers[303] = $250
registers[304] = $250
registers[305] = $250
registers[306] = $250
registers[307] = $250
registers[308] = $250
registers[309] = $250
registers[310] = $250
registers[311] = $250
registers[312] = $250
registers[313] = $250
registers[314] = $250
registers[315] = $250
registers[316] = $250
registers[317] = $250
registers[318] = $250
registers[319] = $250
registers[320] = $250
registers[321] = $250
registers[322] = $250
registers[323] = $250
registers[324] = $250
registers[325] = $250
registers[326] = $250
registers[327] = $250
registers[328] = $250
registers[329] = $250
registers[330] = $250
registers[331] = $250
registers[332] = $250
registers[333] = $250
registers[334] = $250
registers[335] = $250
registers[336] = $250
registers[337] = $250
registers[338] = $250
registers[339] = $250
registers[340] = $250
registers[341] = $250
registers[342] = $250
registers[343] = $250
registers[344] = $250
registers[345] = $250
registers[346] = $250
registers[347] = $250
registers[348] = $250
registers[349] = $250
registers[350] = $250
registers[351] = $250
registers[352] = $250
registers[353] = $250
registers[354] = $250
registers[355] = $250
registers[356] = $250
registers[357] = $250
registers[358] = $250
registers[359] = $250
registers[360] = $250
registers[361] = $250
registers[362] = $250
registers[363] = $250
registers[364] = $250
registers[365] = $250
registers[366] = $250
registers[367] = $250
registers[368] = $250
registers[369] = $250
registers[370] = $250
registers[371] = $250
registers[372] = $250
registers[373] = $250
registers[374] = $250
registers[375] = $250
registers[376] = $250
registers[377] = $250
registers[378] = $250
registers[379] = $250
registers[380] = $250
registers[381] = $250
registers[382] = $250
registers[383] = $250
registers[384] = $250
registers[385] = $250
registers[386] = $250
registers[387] = $250
registers[388] = $250
registers[389] = $250
registers[390] = $250
registers[391] = $250
registers[392] = $250
registers[393] = $250
registers[394] = $250
registers[395] = $250
registers[396] = $250
registers[397] = $250
registers[398] = $250
registers[399] = $250
registers[400] = $250
registers[401] = $250
registers[402] = $250
registers[403] = $250
registers[404] = $250
registers[405] = $250
registers[406] = $250
registers[407] = $250
registers[408] = $250
registers[409] = $250
registers[410] = $250
registers[411] = $250
registers[412] = $250
registers[413] = $250
registers[414] = $250
registers[415] = $250
registers[416] = $250
registers[417] = $250
registers[418] = $250
registers[419] = $250
registers[420] = $250
registers[421] = $250
registers[422] = $250
registers[423] = $250
registers[424] = $250
registers[425] = $250
registers[426] = $250
registers[427] = $250
registers[428] = $250
registers[429] = $250
registers[430] = $250
registers[431] = $250
registers[432] = $250
registers[433] = $250
registers[434] = $250
registers[435] = $250
registers[436] = $250
registers[437] = $250
registers[438] = $250
registers[439] = $250
registers[440] = $250
registers[441] = $250
registers[442] = $250
registers[443] = $250
registers[444] = $250
registers[445] = $250
registers[446] = $250
registers[447] = $250
registers[448] = $250
registers[449] = $250
registers[450] = $250
registers[451] = $250
registers[452] = $250
registers[453] = $250
registers[454] = $250
registers[455] = $250
registers[456] = $250
registers[457] = $250
registers[458] = $250
registers[459] = $250
registers[460] = $250
registers[461] = $250
registers[462] = $250
registers[463] = $250
registers[464] = $250
registers[465] = $250
registers[466] = $250
registers[467] = $250
registers[468] = $250
registers[469] = $250
registers[470] = $250
registers[471] = $250
registers[472] = $250
registers[473] = $250
registers[474] = $250
registers[475] = $250
registers[476] = $250
registers[477] = $250
registers[478] = $250
registers[479] = $250
registers[480] = $250
registers[481] = $250
registers[482] = $250
registers[483] = $250
registers[484] = $250
registers[485] = $250
registers[486] = $250
registers[487] = $250
registers[488] = $250
registers[489] = $250
registers[490] = $250
registers[491] = $250
registers[492] = $250
registers[493] = $250
registers[494] = $250
registers[495] = $250
registers[496] = $250
registers[497] = $250
registers[498] = $250
registers[499] = $250
registers[500] = $250
registers[501] = $250
registers[502] = $250
registers[503] = $250
registers[504] = $250
registers[505] = $250
registers[506] = $250
registers[507] = $250
registers[508] = $250
registers[509] = $250
registers[510] = $250
registers[511] = $250
registers[512] = $250
registers[513] = $250
registers[514] = $250
registers[515] = $250
registers[516] = $250
registers[517] = $250
registers[518] = $250
registers[519] = $250
registers[520] = $250
registers[521] = $250
registers[522] = $250
registers[523] = $250
registers[524] = $250
registers[525] = $250
registers[526] = $250
registers[527] = $250
registers[528] = $250
registers[529] = $250
registers[530] = $250
registers[531] = $250
registers[532] = $250
registers[533] = $250
registers[534] = $250
registers[535] = $250
registers[536] = $250
registers[537] = $250
registers[538] = $250
registers[539] = $250
registers[540] = $250
registers[541] = $250
registers[542] = $250
registers[543] = $250
registers[544] = $250
registers[545] = $250
registers[546] = $250
registers[547] = $250
registers[548] = $250
registers[549] = $250
registers[550] = $250
registers[551] = $250
registers[552] = $250
registers[553] = $250
registers[554] = $250
registers[555] = $250
registers[556] = $250
registers[557] = $250
registers[558] = $250
registers[559] = $250
registers[560] = $250
registers[561] = $250
registers[562] = $250
registers[563] = $250
registers[564] = $250
registers[565] = $250
registers[566] = $250
registers[567] = $250
registers[568] = $250
registers[569] = $250
registers[570] = $250
registers[571] = $250
registers[572] = $250
registers[573] = $250
registers[574] = $250
registers[575] = $250
registers[576] = $250
registers[577] = $250
registers[578] = $250
registers[579] = $250
registers[580] = $250
registers[581] = $250
registers[582] = $250
registers[583] = $250
registers[584] = $250
registers[585] = $250
registers[586] = $250
registers[587] = $250
registers[588] = $250
registers[589] = $250
registers[590] = $250
registers[591] = $250
registers[592] = $250
registers[593] = $250
registers[594] = $250
registers[595] = $250
registers[596] = $250
registers[597] = $250
registers[598] = $250
registers[599] = $250
registers[600] = $250
registers[601] = $250
registers[602] = $250
registers[603] = $250
registers[604] = $250
registers[605] = $250
registers[606] = $250
registers[607] = $250
registers[608] = $250
registers[609] = $250
registers[610] = $250
registers[611] = $250
registers[612] = $250
registers[613] = $250
registers[614] = $250
registers[615] = $250
registers[616] = $250
registers[617] = $250
registers[618] = $250
registers[619] = $250
registers[620] = $250
registers[621] = $250
registers[622] = $250
registers[623] = $250
registers[624] = $250
registers[625] = $250
registers[626] = $250
registers[627] = $250
registers[628] = $250
registers[629] = $250
registers[630] = $250
registers[631] = $250
registers[632] = $250
registers[633] = $250
registers[634] = $250
registers[635] = $250
registers[636] = $250
registers[637] = $250
registers[638] = $250
registers[639] = $250
registers[640] = $250
registers[641] = $250
registers[642] = $250
registers[643] = $250
registers[644] = $250
registers[645] = $250
registers[646] = $250
registers[647] = $250
registers[648] = $250
registers[649] = $250
registers[650] = $250
registers[651] = $250
registers[652] = $250
registers[653] = $250
registers[654] = $250
registers[655] = $250
registers[656] = $250
registers[657] = $250
registers[658] = $250
registers[659] = $250
registers[660] = $250
registers[661] = $250
registers[662] = $250
registers[663] = $250
registers[664] = $250
registers[665] = $250
registers[666] = $250
registers[667] = $250
registers[668] = $250
registers[669] = $250
registers[670] = $250
registers[671] = $250
registers[672] = $250
registers[673] = $250
registers[674] = $250
registers[675] = $250
registers[676] = $250
registers[677] = $250
registers[678] = $250
registers[679] = $250
registers[680] = $250
registers[681] = $250
registers[682] = $250
registers[683] = $250
registers[684] = $250
registers[685] = $250
registers[686] = $250
registers[687] = $250
registers[688] = $250
registers[689] = $250
registers[690] = $250
registers[691] = $250
registers[692] = $250
registers[693] = $250
registers[694] = $250
registers[695] = $250
registers[696] = $250
registers[697] = $250
registers[698] = $250
registers[699] = $250
registers[700] = $250
registers[701] = $250
registers[702] = $250
registers[703] = $250
registers[704] = $250
registers[705] = $250
registers[706] = $250
registers[707] = $250
registers[708] = $250
registers[709] = $250
registers[710] = $250
registers[711] = $250
registers[712] = $250
registers[713] = $250
registers[714] = $250
registers[715] = $250
registers[716] = $250
registers[717] = $250
registers[718] = $250
registers[719] = $250
registers[720] = $250
registers[721] = $250
registers[722] = $250
registers[723] = $250
registers[724] = $250
registers[725] = $250
registers[726] = $250
registers[727] = $250
registers[728] = $250
registers[729] = $250
registers[730] = $250
registers[731] = $250
registers[732] = $250
registers[733] = $250
registers[734] = $250
registers[735] = $250
registers[736] = $250
registers[737] = $250
registers[738] = $250
registers[739] = $250
registers[740] = $250
registers[741] = $250
registers[742] = $250
registers[743] = $250
registers[744] = $250
registers[745] = $250
registers[746] = $250
registers[747] = $250
registers[748] = $250
registers[749] = $250
registers[750] = $250
registers[751] = $250
registers[752] = $250
registers[753] = $250
registers[754] = $250
registers[755] = $250
registers[756] = $250
registers[757] = $250
registers[758] = $250
registers[759] = $250
registers[760] = $250
registers[761] = $250
registers[762] = $250
registers[763] = $250
registers[764] = $250
registers[765] = $250
registers[766] = $250
registers[767] = $250
registers[768] = $250
registers[769] = $250
registers[770] = $250
registers[771] = $250
registers[772] = $250
registers[773] = $250
registers[774] = $250
registers[775] = $250
registers[776] = $250
registers[777] = $250
registers[778] = $250
registers[779] = $250
registers[780] = $250
registers[781] = $250
registers[782] = $250
registers[783] = $250
registers[784] = $250
registers[785] = $250
registers[786] = $250
registers[787] = $250
registers[788] = $250
registers[789] = $250
registers[790] = $250
registers[791] = $250
registers[792] = $250
registers[793] = $250
registers[794] = $250
registers[795] = $250
registers[796] = $250
registers[797] = $250
registers[798] = $250
registers[799] = $250
registers[800] = $250
registers[801] = $250
registers[802] = $250
registers[803] = $250
registers[804] = $250
registers[805] = $250
registers[806] = $250
registers[807] = $250
registers[808] = $250
registers[809] = $250
registers[810] = $250
registers[811] = $250
registers[812] = $250
registers[813] = $250
registers[814] = $250
registers[815] = $250
registers[816] = $250
registers[817] = $250
registers[818] = $250
registers[819] = $250
registers[820] = $250
registers[821] = $250
registers[822] = $250
registers[823] = $250
registers[824] = $250
registers[825] = $250
registers[826] = $250
registers[827] = $250
registers[828] = $250
registers[829] = $250
registers[830] = $250
registers[831] = $250
registers[832] = $250
registers[833] = $250
registers[834] = $250
registers[835] = $250
registers[836] = $250
registers[837] = $250
registers[838] = $250
registers[839] = $250
registers[840] = $250
registers[841] = $250
registers[842] = $250
registers[843] = $250
registers[844] = $250
registers[845] = $250
registers[846] = $250
registers[847] = $250
registers[848] = $250
registers[849] = $250
registers[850] = $250
registers[851] = $250
registers[852] = $250
registers[853] = $250
registers[854] = $250
registers[855] = $250
registers[856] = $250
registers[857] = $250
registers[858] = $250
registers[859] = $250
registers[860] = $250
registers[861] = $250
registers[862] = $250
registers[863] = $250
registers[864] = $250
registers[865] = $250
registers[866] = $250
registers[867] = $250
registers[868] = $250
registers[869] = $250
registers[870] = $250
registers[871] = $250
registers[872] = $250
registers[873] = $250
registers[874] = $250
registers[875] = $250
registers[876] = $250
registers[877] = $250
registers[878] = $250
registers[879] = $250
registers[880] = $250
registers[881] = $250
registers[882] = $250
registers[883] = $250
registers[884] = $250
registers[885] = $250
registers[886] = $250
registers[887] = $250
registers[888] = $250
registers[889] = $250
registers[890] = $250
registers[891] = $250
registers[892] = $250
registers[893] = $250
registers[894] = $250
registers[895] = $250
registers[896] = $250
registers[897] = $250
registers[898] = $250
registers[899] = $250
registers[900] = $250
registers[901] = $250
registers[902] = $250
registers[903] = $250
registers[904] = $250
registers[905] = $250
registers[906] = $250
registers[907] = $250
registers[908] = $250
registers[909] = $250
registers[910] = $250
registers[911] = $250
registers[912] = $250
registers[913] = $250
registers[914] = $250
registers[915] = $250
registers[916] = $250
registers[917] = $250
registers[918] = $250
registers[919] = $250
registers[920] = $250
registers[921] = $250
registers[922] = $250
registers[923] = $250
registers[924] = $250
registers[925] = $250
registers[926] = $250
registers[927] = $250
registers[928] = $250
registers[929] = $250
registers[930] = $250
registers[931] = $250
registers[932] = $250
registers[933] = $250
registers[934] = $250
registers[935] = $250
registers[936] = $250
registers[937] = $250
registers[938] = $250
registers[939] = $250
registers[940] = $250
registers[941] = $250
registers[942] = $250
registers[943] = $250
registers[944] = $250
registers[945] = $250
registers[946] = $250
registers[947] = $250
registers[948] = $250
registers[949] = $250
registers[950] = $250
registers[951] = $250
registers[952] = $250
registers[953] = $250
registers[954] = $250
registers[955] = $250
registers[956] = $250
registers[957] = $250
registers[958] = $250
registers[959] = $250
registers[960] = $250
registers[961] = $250
registers[962] = $250
registers[963] = $250
registers[964] = $250
registers[965] = $250
registers[966] = $250
registers[967] = $250
registers[968] = $250
registers[969] = $250
registers[970] = $250
registers[971] = $250
registers[972] = $250
registers[973] = $250
registers[974] = $250
registers[975] = $250
registers[976] = $250
registers[977] = $250
registers[978] = $250
registers[979] = $250
registers[980] = $250
registers[981] = $250
registers[982] = $250
registers[983] = $250
registers[984] = $250
registers[985] = $250
registers[986] = $250
registers[987] = $250
registers[988] = $250
registers[989] = $250
registers[990] = $250
registers[991] = $250
registers[992] = $250
registers[993] = $250
registers[994] = $250
registers[995] = $250
registers[996] = $250
registers[997] = $250
registers[998] = $250
registers[999] = $250
time = 126002
